lib/internal/process/task_queues.js

```diff
@@ -41,6 +42,10 @@ const {
 
 const { AsyncResource } = require('async_hooks');
 
+const AsyncContextFrame = require('internal/async_context_frame');
+
+const async_context_frame = Symbol('asyncContextFrame');
```

Comment:
```suggestion
const async_context_frame = Symbol('kAsyncContextFrame');
```

Not sure but I think there was some naming guideline somewhere. But maybe I'm wrong so feel free to ignore.

if you go for the change there are a few similar places.